docs/modules/ROOT/pages/how-tos/testing/integration/periodic-integration-tests.adoc

```diff
@@ -0,0 +1,134 @@
+= Triggering Periodic Integration Tests
+
+In {ProductName}, you can run integration tests on a schedule using Kubernetes CronJobs.
+
+== Procedure
+
+To trigger a periodic Integration Test, complete the following steps:
+
+. Create an integration test by following the xref:how-tos/testing/integration/creating.adoc[Creating a integration test guide].
+
+[NOTE]
+====
+If the Integration Test is designed to run periodically rather than during Pull Request checks or post-merge, you need to add the *disabled* xref:how-tos/testing/integration/choosing-contexts.adoc[context] to the Integration Test.
```

Comment:
```suggestion
If the Integration Test is designed to only run periodically rather than during Pull Request checks or post-merge, you need to add the *disabled* xref:how-tos/testing/integration/choosing-contexts.adoc[context] to the Integration Test.
```